docs/tla/BackupCol.tla

```diff
@@ -0,0 +1,88 @@
+(* MIT License Copyright (c) 2020 CQFN
+ *    https://github.com/cqfn/degitx/blob/master/LICENSE
+ *
+ * The module shows a conflict in the 3-phase commit protocol with the backup transaction manager.
```

Comment:
This module has only 4 states (working, prepared, committed and aborted), so it's 2PC, not 3PC, isn't it?

Reply:
This is part of 3PC which is interesting for us. Fork may rise potential issue is placed in "working" state only.